src/core/Classes/Author_Editor.php

```diff
@@ -625,4 +626,28 @@ public static function admin_notices()
             echo '</div>';
         }
     }
+
+    /**
+     * Author term mapped limit validation
+	 *
+	 * @param string|WP_Error $term     The term name to add, or a WP_Error object if there's an error.
+	 * @param string          $taxonomy Taxonomy slug.
+     * 
+     * @return array|WP_Error
+     */
+    public static function filter_pre_insert_term($term, $taxonomy)
```

Comment:
This is working well. But we need to make an additional verification for the Author URL field. We can't have two authors using the same Author URL, and we can't have an author using the Author URL of another user (not only author), if not set as mapped to that user.

Reply:
I'll only be checking for user with slug and skip term with same slug as the check is already existing. It's impossible to add author with same slug. So, will be adding a check for user